public/index.html

```diff
@@ -36,27 +36,30 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>thenewboston</title>
+    <title>LeapChain</title>
     <meta name="theme-color" content="#004ba0" data-react-helmet="true" />
-    <meta property="og:url" content="https://thenewboston.com" />
+    <meta property="og:url" content="https://leapchain.surge.sh" />
```

Comment:
this one needs to be updated to the one @mrbusysky deployed, the one in surge is temporary

Reply:
@mrbusysky can you comment for the updated domain? - thx